### PR TITLE
🌱 Clarify our goals for CAPV releases related to CAPI, CPI and Kubernetes releases

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,11 @@ Basically:
 - 4 Kubernetes minor releases for the management cluster (N - N-3)
 - 6 Kubernetes minor releases for the workload cluster (N - N-5)
 
-**NOTE:** As the versioning for this project is tied to the versioning of Cluster API, future modifications to this policy may be made to more closely align with other providers in the Cluster API
-ecosystem.
+**NOTES:**
+* As the versioning for this project is tied to the versioning of Cluster API, future modifications to this
+  policy may be made to more closely align with other providers in the Cluster API ecosystem.
```
Comment (fabriziopandini) [inline README.md:54]: Not blocking, but I will drop this sentence (It doesn't seem to add much value to me)

File: README.md [hunk 2]
```diff
@@ -49,8 +49,11 @@ Basically:
...
+* We aim to cut a CAPV minor release one week after the corresponding CAPI minor release is out.
```
Comment (fabriziopandini) [inline README.md:55]:
```suggestion
* We aim to cut a CAPV minor release approximately one week after the corresponding CAPI minor release is out.
```

File: README.md [hunk 3]
```diff
@@ -49,8 +49,11 @@ Basically:
...
+* We aim to cut a CAPV minor or patch release with support for a new Kubernetes minor version 3 days after releases for CAPI and CPI that support the new Kubernetes version are available.
```
Comment (fabriziopandini) [inline README.md:56]:
```suggestion
* We aim to cut a CAPV minor or patch release with support for a new Kubernetes minor version approximately 3 business days after releases for CAPI and CPI that support the new Kubernetes version are available.
```

**Review by fabriziopandini — COMMENTED** [review]
Thanks for taking the time to formalize those practices/policies!
Just nits from my side

File: README.md (hunk 1 above)
Comment (sbueringer) [reply README.md:54]: Makes sense. I wasn't entirely sure if I should just drop it :D

**Comment by sbueringer** [comment]
Thx for the review, fixed

**Pull request merged**